components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/util/OrganizationSwitchGrantConstants.java

```diff
@@ -27,6 +27,8 @@ public class OrganizationSwitchGrantConstants {
 
     public static final String TOKEN_BINDING_REFERENCE = "tokenBindingReference";
 
+    public static final String ERROR_CODE_APPLICATION_NOT_FOUND = "ORG-60037";
```

Comment:
Can't we reuse the error code defined in org mgt component appending the error code prefix(get the prefix also from existing constant if possible )

Reply:
Fixed